pkg/sass-parser/lib/src/expression/boolean.test.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {BooleanExpression} from '../..';
+import * as utils from '../../../test/utils';
+
+describe('a boolean expression', () => {
+  let node: BooleanExpression;
+
+  describe('true', () => {
+    function describeNode(
+      description: string,
+      create: () => BooleanExpression
+    ): void {
+      describe(description, () => {
+        beforeEach(() => void (node = create()));
+
+        it('has sassType boolean', () => expect(node.sassType).toBe('boolean'));
+
+        it('is true', () => expect(node.value).toBe(true));
+      });
+    }
+
+    describeNode('parsed', () => utils.parseExpression('true'));
+
+    describeNode(
+      'constructed manually',
+      () =>
+        new BooleanExpression({
+          value: true,
+        })
```

Comment:
Nit:

```suggestion
      () => new BooleanExpression({value: true})
```

Prettier won't do this for you, unfortunately. Also below.